addon/components/frost-fixed-table.js

```diff
@@ -22,14 +22,17 @@ export default Component.extend({
   propTypes: {
     // options
     columns: PropTypes.arrayOf(ColumnPropType),
-    items: ItemsPropType
+    items: ItemsPropType,
```

Comment:
It's kind of *useless* not *useful*, but I'm guessing that's what you meant to say.